Dockerfile

```
@@ -17,7 +17,7 @@ FROM python:3.9.6-buster
 
 COPY . /service
 WORKDIR /service
-RUN pip install -r requirements.txt
+RUN python setup.py install
```

```suggestion
RUN pip install .
```
Sorry @deepakunni3, this was wrong in the template repository but is now fixed.

---

No worries! thanks for the pointer 👍